docs/fleet/provision.md

```diff
@@ -23,7 +23,7 @@ The Viam Agent:
 - Automatically connects to a pre-configured WiFi network, or creates its own wireless hotspot if no pre-configured WiFi network is detected.
 - Installs `viam-server` as a static binary, removing the need to perform any library linking or dependency installation during first-time setup.
   You can also use a custom build of `viam-server`, if needed.
-- Provides automatic updates for `viam-server`, the agent itself, and any configured subsystems (such as the Agent Provisioning subsystem).
+- Provides automatic updates for `viam-server`, the agent itself, and any configured functionalities it controls, such as Agent Provisioning.
```

Comment:
The sub-components that can be optionally used with the Agent are still called subsystems, both in the raw JSON (i.e.  `agent.viam-server.disable_subsystem: False` ) and on their [GH Readmes](https://github.com/viamrobotics/agent-provisioning)).

I would retain the original phrasing here.